include/gauge_field_order.h

```diff
@@ -945,4 +945,29 @@ namespace quda {
   };
 
 
-}
+  
+  // Use traits to reduce the template explosion
+  template<typename ,QudaReconstructType,int N=18> struct gauge_mapper { };
```

Comment:
There might be reason for the comma before `QudaReconstructType` but that seem strange to me. I have not yet asked the compiler for comments.